python/pyarrow/table.pxi

```diff
@@ -2010,6 +2010,37 @@ cdef class Table(_PandasConvertible):
 
         return result
 
+    def to_reader(self, max_chunksize=None, **kwargs):
+        """
+        Convert a Table to RecordBatchReader
+
+        Returns
+        -------
+        RecordBatchReader        
+        """
+        cdef:
+            shared_ptr[CRecordBatchReader] c_reader
+            RecordBatchReader reader
+            shared_ptr[TableBatchReader] t_reader
+        t_reader = make_shared[TableBatchReader](self.sp_table)
+
+        if 'chunksize' in kwargs:
+            max_chunksize = kwargs['chunksize']
+            msg = ('The parameter chunksize is deprecated for '
+                   'pyarrow.Table.to_batches as of 0.15, please use '
+                   'the parameter max_chunksize instead')
+            warnings.warn(msg, FutureWarning)
+
+        if max_chunksize is not None:
+            c_max_chunksize = max_chunksize
```

Comment:
nit: I don't think this extra assignment does anything (though maybe Cython requires it for some reason)

Reply:
Removed the extra assigment